app/controllers/part_payments_controller.rb

```diff
@@ -36,6 +36,14 @@ def confirmation
     @result = Views::PartPayment::Result.new(part_payment)
   end
 
+  def return_letter
+    application_overview
+  end
+
+  def return_application
+    redirect_to root_path if ResolverService.new(part_payment, current_user).resolve('return')
```

Comment:
If the `ResolverService` fails to resolve, this will raise an error because the template is missing.